client/src/components/Ticketing/ticketingmanager/AdminPurchase/AdminCheckout.tsx

```diff
@@ -86,9 +90,10 @@ export default function CheckoutPage(): ReactElement {
                 Complete Order
               </div>
               <AdminCompleteOrderForm
-                disabled={cartItems.length === 0}
+                disabled={false} // {cartItems.length === 0}
```

Comment:
I don't think the `disabled` prop does anything so it can be removed

Reply:
removed